app/commands/get_user_review_request_metrics.rb

```diff
@@ -0,0 +1,35 @@
+class GetUserReviewRequestMetrics < PowerTypes::Command.new(:github_user, :limit_month)
+  def perform
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Metrics/AbcSize: Assignment Branch Condition size for perform is too high. [<13, 23, 4> 26.72/25]